client/Components/SingleCohort/SubmittedStretches.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+import { connect } from 'react-redux'
+import { getAllCohortUsers } from '../../store/cohort-users/actions'
+import List from '@material-ui/core/List'
+import Typography from '@material-ui/core/Typography'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemText from '@material-ui/core/ListItemText'
+import Collapse from '@material-ui/core/Collapse'
+import ExpandLess from '@material-ui/icons/ExpandLess'
+import ExpandMore from '@material-ui/icons/ExpandMore'
+
+const SubmittedStretches = ({ student, stretchAnswers, stretches, cohortUsers }) => {
+
+    const [studentsExpanded, toggleStudents] = useState(false)
+
+    const handleClickStudents = () => toggleStudents(!studentsExpanded)
+
+    const studentCohortUserId = cohortUsers.find(user => user.userId === student.id)
```

Comment:
So right now we are allowing a student to be in multiple cohorts so this could end up not finding the right cohort. But I'm going to change StretchAnswers to be associated to a stretchcohort to help with some of the logic elsewhere, which will require changing this a bit so I can make a change then.